addons/addon-base-workflow/packages/base-workflow-core/lib/workflow/workflow-trigger-service.js

```diff
@@ -84,7 +84,17 @@ async function triggerStepFunctions({ instance, meta, input }) {
     name,
   };
 
-  const data = await sf.startExecution(params).promise();
+  let data = {};
+  try {
+    data = await sf.startExecution(params).promise();
+  } catch (e) {
+    throw this.boom.internalError(
+      `Step Function could not start execution for State Machine ${stateMachineArn} with params ${JSON.stringify(
+        params,
+      )}. Error code: ${e.code}`,
+      false,
```

Comment:
optional: Why not also display the error message on SWB UI?

```suggestion
      true,
```

Reply:
The security ticket ask us to not reveal error codes in the response. Setting this value to false doesn't show the error code in the API response, but still logs it in Cloudwatch Logs. Logging the error in CWL will help the developer debug the issue.